src/structures/Chat.js

```diff
@@ -192,6 +192,22 @@ class Chat extends Base {
         return messages.map(m => new Message(this.client, m));
     }
 
+    /**
+     * Fetches a message from the chat.
+     * @param {String} messageId The Message id of message looking for.
+     * @returns {Promise<Message>}
+     */
+    async getMessageById(messageId) {
+        let message = await this.client.pupPage.evaluate(async (chatId, messageId) => {
+            const chat = window.Store.Chat.get(chatId);
+            const msg = chat.msgs.models.find(m => m.id.id === messageId);
+            return window.WWebJS.getMessageModel(msg);
+
+        }, this.id._serialized, messageId);
+
+        return new Message(this.client, message);
+    }
```

Comment:
You might want to take a look at `window.Store.Msg.get(msgId)`. It allows you to retrive msg by id and no need to getChat operation